test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/Connection/ConnectionServiceTests.cs

```diff
@@ -124,7 +124,8 @@ public async Task GetCurrentConnectionStringTest()
             var requestParams = new GetConnectionStringParams()
             {
                 OwnerUri = result.ConnectionInfo.OwnerUri,
-                IncludePassword = false
+                IncludePassword = false,
+                IncludeApplicationName = true
```

Comment:
Seems like a good opportunity to add a test for not having the application name as well 😄